i2c/README.md

```diff
@@ -185,3 +191,129 @@ SensorLightPushButton:
             - 'DIM'
             - 'STOP'
 ```
+
+## `i2c.pwm.PwmHatColorLED`
+
+Commands 1, 3 or 4 channels on the [Adafruit PWM HAT](https://learn.adafruit.com/adafruit-16-channel-pwm-servo-hat-for-raspberry-pi) to control a white, RGB or RGBW LED.
+A received command will be sent back on all configured connections to the configured return topic, to keep them up to date.
+
+### Dependencies
+
+This actuator communicates via the i2c interface, therefore the GPIO 2 and 3 should be free and cannot get accessed directly e. g. with a RpiGPIOSensor.
+The user running sensor_reporter must be in the `i2c` group to have access to the i2c interface.
+Also the library `smbus2` must be installed to make this actuator work.
+
+To enable the i2c interface open the Raspberry-Pi configuration via:
```

Comment:
Not a request, just a question. Could this be implemented as part of the install scripts? Obviously calling raspi-config isn't going to work but I think all that does is set a flag in a file somewhere.